### DISCUSSION TITLE
Waterless Clouds in Analysis

### DISCUSSION THREAD

**Discussion by csteele2** [NOAA-GSL/GSL-Regional-Model-Forum#47] [Q&A] [answered]
I have been attempting to use HRRR Analysis to build a dataset of some orographic clouds, specifically orographic clouds containing supercooled liquid water. I know cloud phase is a tricky thing to model, but I have been mostly impressed at the quality of HRRR analysis when comparing to satellite RGBs. However, I occasionally get an odd case where there seems to be data "cut out" or a hole in my derived fields. In poking at the HRRR data, these cases appear to have clouds (via cloud fraction), but don't contain any liquid or ice water mass. I can totally see this being a product of how something like a cloud mask is ingested into DA, but just wanted to double check to make sure. I know this is a really specialized application of HRRR analysis, but any thoughts on this?

Example:
![HRRR_analysis_4P_hole](https://github.com/user-attachments/assets/b918263f-9757-4acc-ba17-f41e58a57300)

A positive example (so I know I am probably doing things right):
![HRRR_analysis_4P](https://github.com/user-attachments/assets/8090bd36-f763-4a94-8c28-35cd6f310e72)

**Comment by EricJames-NOAA** [comment]
@csteele2 sorry for the very late response, and thanks for the question.  You're right that there are definitely some artifacts in the HRRR cloud analysis fields because of the non-variational cloud analysis that we carry out each hour.  You can find a description of the algorithm here:

https://doi.org/10.1175/MWR-D-20-0319.1

In particular, cloud building based on METAR observations creates circular areas of cloud in the initial conditions, while satellite cloud clearing can also produce sharp cut-offs in the cloud fields.  Sometimes the cloud building regions can have a different shape if there is terrain in the way.  

Hopefully this is helpful!  It sounds like an interesting analysis you are doing.

**Comment by EricJames-NOAA** [comment, marked as answer]
@csteele2 a colleague has pointed out the mismatch between a nonzero cloud fraction but zero cloud or ice mixing ratio could be due to subgrid clouds.  These are not modified by the cloud analysis procedure, but subgrid clouds can exist coming out of the 1-h pre-forecast.  

Just another idea for your analysis.
